### ISSUE TITLE
Repeated pandas FutureWarning messages

### ISSUE THREAD

**Issue by AlexaDiNicola** [dereneaton/ipyrad#544]
Throughout steps 1-3 (possibly farther, but I haven't gotten past that point yet), my ipyrad output is being overrun with the following warning message:

```
  from saved path: /Volumes/[more path]/ipyrad-breweri/breweri2023.json
/Users/[more path]/miniconda3/envs/ipyrad/lib/python3.10/site-packages/ipyrad/core/assembly.py:266: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\s+'`` instead
  bdf = pd.read_csv(barcodefile[0], header=None, delim_whitespace=1)
```

It seems to be showing up in many of the places where there'd otherwise be an empty line output, and it's getting harder to fish the actual information out from a block of like 6 warning repeats. I'm running ipyrad 0.9.93 and miniconda 23.11.0. Any idea what's going on here or how I can suppress that? (All I'm finding are ways to suppress it in the actual Python code, which I really don't want to mess with, haha.)

**Comment by isaacovercast** [comment]
Hi Alexa,

Thanks for reporting this, there was a change in the new version of pandas. I fixed this and pushed a new version **0.9.94** which will be up on bioconda in the next day or so. If you get a chance to update to 0.9.94 and verify that it is working for you that would be great. The new package could be up Sunday, but the weekend sometimes slows down bioconda package deployment, so it could be Monday.

**Comment by AlexaDiNicola** [comment]
Oh, fantastic -- thank you! Will do.
